index.js

```diff
@@ -1,3 +1,38 @@
+var _headerData = function(appConfig, options) {
```

Comment:
Can you move this into a utility function?  I would like to share this with a future command that will emit the header value to the console...